README.rst

```diff
@@ -254,7 +254,7 @@ _____________
 
 .. code-block:: bash
 
-    docker run -d -p 5672:5672 -p 15672:15672 rabbitmq:3-management
+    docker run -d -p 5671:5671 -p 5672:5672 -p 15671:15671 -p 15672:15672 mosquito/aiormq-rabbitmq
```

Comment:
what's wrong with official docker image?

Reply:
Nothing, it just isn't the image the make file uses. This just updates the contributor documentation to match what happens in Github Actions.

Reply:
Well, it is wrong in the sense that it is not entirely what the test suite expects.